### PR TITLE
Modulo 3, inciso 4

### PR DESCRIPTION
**Pull request by perez-alex**
Agregado soporte para ActiveAndroid

### CODE REVIEW

File: retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/Image.java [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
 
+import java.util.List;
+
 /**
  * Created by aperez on 24/01/17.
  */
 
-public class Image {
+@Table(name = "Images")
+public class Image extends Model{
```
Comment (nacho-globant) [inline retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/Image.java]: Create another class for this. The Image class is used as response to a retrofit service.

File: retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java [hunk 2]
```diff
@@ -47,4 +53,20 @@ public void onFailure(Call<LatestResponse> call, Throwable t) {
             }
         });
     }
+
+    public List<Image> getSavedImages() {
+        return Image.getAll();
+    }
+
+    public void saveImages(List<Image> images) {
+        ActiveAndroid.beginTransaction();
```
Comment (nacho-globant) [inline retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java]: As a good practice. You could create another class, to manage the Image table. For example, ImageRepository. Not necessary to apply change, only a sugerency

File: retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java [hunk 3]
```diff
@@ -30,7 +32,11 @@ public ResponseModel(Bus bus) {
     public void getLatestImages(){
         Retrofit retrofit = new Retrofit.Builder()
```
Comment (nacho-globant) [inline retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java]: As a good practice. You could create another class, to manage the Retrofit services. For example, Services. Not necessary to apply change, only a sugerency

**Comment by nacho-globant** [comment]
Added some comments

**Comment by perez-alex** [comment]
Revisado!

File: retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/db/StoredImage.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+    public String getLargeUrl() {
+        return largeUrl;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public static List<StoredImage> getAll() {
```
Comment (nacho-globant) [inline retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/db/StoredImage.java]: This method should be in the ImageRepository.

**Pull request merged**